src/components/CardList/index.jsx

```diff
@@ -1,17 +1,16 @@
 import React, { useEffect, useState } from 'react';
```

Comment:
Let's try to avoid using document.getElementById, that's the vanilla js way to handle that, in react you should use the hook `useRef`